mne/viz/eyetracking/heatmap.py

```diff
@@ -0,0 +1,185 @@
+# Authors: Scott Huberty <seh33@uw.edu>
+#
+# License: BSD-3-Clause
+
+import numpy as np
+from scipy.ndimage import gaussian_filter
+
+
+from ..utils import plt_show
+from ...utils import _validate_type, logger, fill_doc
+
+
+@fill_doc
+def plot_gaze(
+    epochs,
+    width,
+    height,
+    bin_width=10,
+    sigma=1,
+    cmap=None,
+    alpha=None,
+    vmin=None,
+    vmax=None,
+    axes=None,
+    show=True,
+):
+    """Plot a heatmap of eyetracking gaze data.
+
+    Parameters
+    ----------
+    epochs : mne.Epochs
+        The :class:`~mne.Epochs` object containing eyegaze channels.
+    width : int
+        The width dimension of the plot canvas. For example, if the eyegaze data units
+        are pixels, and the participant screen resolution was 1920x1080, then the width
+        should be 1920.
+    height : int
+        The height dimension of the plot canvas. For example, if the eyegaze data units
+        are pixels, and the participant screen resolution was 1920x1080, then the height
+        should be 1080.
+    bin_width : int
+        The number of eyegaze units per square bin that are used to create the heatmap.
+        Default is 10, meaning that if the eyegaze units are pixels, each bin is 10x10
+        pixels. See the appendix section of :ref:`tut-eyetrack-heatmap` for more
+        detail.
```

Comment:
Do we need this? I don't see much point as this risks spatial aliasing (it's like downsampling data without lowpassing). To me if you want a smooth image the right thing to do is smoothing using a Gaussian or some other kernel. And maybe only once you've done that you could call this `decim` or something... but do people really want/need this for most use cases? (By omitting it we avoid issues with having to worry about spatial aliasing altogether...)

Reply:
I probably missed the discussion :(

But to me if your screen is 1920x1080 you should just smooth/convolve with some appropriately smooth kernel if you want to spread the activation. Make the kernel a 10x10 boxcar if you want. But I don't see an advantage to decimating -- seems to me like the result of the operation should still be some 1920x1080 set of values that gets `imshow`ed. It will look better and smoother and we don't have to worry about spatial aliasing

Reply:
I checked in with @larsoner and @drammock today after you all had discussed it on discord, I think we are all thinking that we don't actually need the bin_width parameter. I'll update this PR and we can review and confirm!